adafruit_ssd1306.py

```diff
@@ -63,7 +66,7 @@ class _SSD1306:
     """Base class for SSD1306 display driver"""
     #pylint: disable-msg=too-many-arguments
     #pylint: disable-msg=too-many-instance-attributes
-    def __init__(self, framebuffer, width, height, external_vcc, reset):
+    def __init__(self, framebuffer, width, height, *, external_vcc, reset):
```

Comment:
since reset is an optional keyword does it default to None? if not should it be explicitly set to None as the default?